README.md

```diff
@@ -1,2 +1,57 @@
-# My-Portfolio
-My personal portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> My Portfolio Project, Module 1: week 1: Day 3
+
+## Built With
+
+- Major languages : HTML & CSS
+- Frameworks : none
+
+## Getting Started
+
+**This project is create by @mrbhi. Go through, to see the content of the project Cheers!**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@mrbhi](https://github.com/mrbhi/my-portfolio-homepage)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the
+[MIT license](https://choosealicense.com/licenses/mit/) - you can set it up
+quickly by
+[using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).
+You can also use [any other license](https://choosealicense.com/licenses/) if
+you wish._
```

Comment:
- [ ] Your readme is descriptive good job :clap:  But please remove this unnecessary informations in order to make it more professional :+1: